tests/pipelines/test_cli.py

```diff
@@ -307,4 +307,5 @@ def test_deploy_pipeline_conflicting_ids(pipelines_api_mock, tmpdir):
 
     result = CliRunner().invoke(cli.deploy_cli, ['--spec', path, '--pipeline-id', "fake"])
     assert result.exit_code == 1
+    assert "ValueError" in result.stdout
```

Comment:
weirdly the CliRunner does not include the ValueError exception as an object so we have to check the output directly

Reply:
Instead of asserting "ValueError", shouldn't we assert some snippet of that helpful error message?

Reply:
added more of the msg